simulator/pkg/debuggablescheduler/debuggable_scheduler.go

```diff
@@ -49,7 +49,7 @@ func CreateOptionForOutOfTreePlugin(outOfTreePluginRegistry runtime.Registry, pl
 
 	var versionedcfg *v1.KubeSchedulerConfiguration
 	var err error
-	if configFile == nil {
+	if configFile == nil || *configFile == "" {
```

Comment:
Can you make a separate PR for this line change?

Reply:
I separate PR.
fix this PR.
[60eff56](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/350/commits/60eff567ca22304a30400c753cc2c9239c70ac5a)
configFIle fix PR.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/351